lib/SILGen/SILGenPattern.cpp

```diff
@@ -1032,17 +1032,25 @@ void PatternMatchEmission::emitWildcardDispatch(ClauseMatrix &clauses,
   assert(!hasGuard || !clauses[row].isIrrefutable());
 
   auto stmt = clauses[row].getClientData<CaseStmt>();
-  ArrayRef<CaseLabelItem> labelItems = stmt->getCaseLabelItems();
-  bool hasMultipleItems = labelItems.size() > 1;
+  bool hasMultipleItems = false;
+  if (stmt->getKind() == StmtKind::Case) {
```

Comment:
Calling `getKind()` is usually not the right thing to do.  Here, you probably want `if (isa<CaseStmt>(S))`.